pyproject.toml

```diff
@@ -29,12 +29,13 @@ documentation = "https://docs.camel-ai.org"
 [tool.poetry.dependencies]
 python = ">=3.8.1,<3.12"
 numpy = "^1"
-openai = "^1.2.3"
```

Comment:
why fix the maximal openai version here?

Reply:
Originally it worked as a temporary fix for issue #474 . And yes the upper bound of openai package version is no longer needed now.